image/verify/fuzz/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "caliptra-image-verify-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = { version = "0.4.6", optional = true }
+afl = { version = "0.13.3", optional = true }
+
+[patch.crates-io]
+byteorder = { git = "https://github.com/benjamindoron/byteorder.git", branch = "struct_aware-1.4.3" }
+zerocopy = { git = "https://github.com/benjamindoron/zerocopy.git", branch = "struct_aware-v0.6.3" }
```

Comment:
I'm good with merging this as-is, but it would be good if you can send upstream PRs so we can eventually depend on upstream.

Reply:
Okay, thanks. The list of changes aren't long, as long as they don't mind being a dependency in fuzzing targets, it should be fine - std is required for cfg(test) or cfg(fuzzing).

Reply:
Does this actually work? Isn't the image crate still using the other version of these crates defined in the workspace?

Reply:
Oh I see, image/verify/fuzz is not part of the workspace, so it can patch anything it wants. Probably worth a comment.

Reply:
https://github.com/chipsalliance/caliptra-sw/issues/681; and commenting done, will be in the next push.

Reply:
I'm not certain it's because this crate is it's own workspace that it can patch anything, because I couldn't define `arbitrary`'s version, as we discussed below, for the root workspace by defining it here. It might just be that any crate build can override dependencies.

Or, I've misunderstood the differences between dependencies in the workspace, and external dependencies.

Reply:
Not sure I understand the question above. I think dependencies here are completely separate from the root workspace because this crate has its own workspace.

Anyway, as long as this works (which it looks like it does) we're good with it.